src/node_file.cc

```diff
@@ -2741,6 +2741,31 @@ static void Chown(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void ChownSync(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  CHECK_GE(args.Length(), 3);
+
+  BufferValue path(env->isolate(), args[0]);
+  CHECK_NOT_NULL(*path);
+  THROW_IF_INSUFFICIENT_PERMISSIONS(
+      env, permission::PermissionScope::kFileSystemWrite, path.ToStringView());
+
+  CHECK(IsSafeJsInt(args[1]));
+  const uv_uid_t uid = static_cast<uv_uid_t>(args[1].As<Integer>()->Value());
+
+  CHECK(IsSafeJsInt(args[2]));
+  const uv_gid_t gid = static_cast<uv_gid_t>(args[2].As<Integer>()->Value());
+
+  uv_fs_t req;
+  auto make = OnScopeLeave([&req]() { uv_fs_req_cleanup(&req); });
```

Comment:
Why is this called `make`? Maybe `cleanup` is a better name?
```suggestion
  auto cleanup = OnScopeLeave([&req]() { uv_fs_req_cleanup(&req); });
```

Reply:
Updated. Thanks! I also had to force push due to rebase.